lib/webpacker/helper.rb

```diff
@@ -9,8 +9,12 @@ module Webpacker::Helper
   # In production mode:
   #   <%= asset_pack_path 'calendar.css' %> # => "/packs/calendar-1016838bab065ae1e122.css"
   def asset_pack_path(name, **options)
-    asset_path(Webpacker::Manifest.lookup(name), **options)
+    path = Webpacker::Configuration.public_output_path
+    file = Webpacker::Manifest.lookup(name)
+    full_path = "#{path}/#{file}"
```

Comment:
@dhh We could configure using the same subdirectory of /public for the webpack-dev-server. Do you have any more details on your service worker example? Would there be new asset helpers?